server/app/views/applicant/ApplicantLayout.java

```diff
@@ -94,18 +96,27 @@ public ApplicantLayout(
       SettingsManifest settingsManifest,
       DeploymentType deploymentType,
       AssetsFinder assetsFinder,
-      PageNotProductionBanner pageNotProductionBanner) {
+      PageNotProductionBanner pageNotProductionBanner,
+      MessagesApi messagesApi) {
     super(viewUtils, settingsManifest, deploymentType, assetsFinder);
     this.layout = layout;
     this.profileUtils = checkNotNull(profileUtils);
     this.languageSelector = checkNotNull(languageSelector);
     this.languageUtils = checkNotNull(languageUtils);
     this.isDevOrStaging = deploymentType.isDevOrStaging();
     this.pageNotProductionBanner = checkNotNull(pageNotProductionBanner);
+    this.messagesApi = checkNotNull(messagesApi);
   }
 
   @Override
   public Content render(HtmlBundle bundle) {
+    // Add the session timeout modals to the bundle if the profile is present
+    Optional<CiviFormProfile> profile =
+        profileUtils.optionalCurrentUserProfile(bundle.getRequest());
+    if (profile.isPresent()) {
+      maybeAddSessionTimeoutModals(bundle, messagesApi.preferred(bundle.getRequest()));
```

Comment:
```suggestion
      addSessionTimeoutModals(bundle, messagesApi.preferred(bundle.getRequest()));
```
(blocking) From above